scanners/a11y.py

```diff
@@ -17,6 +17,7 @@
 def init(environment, options):
     global redirects
     global config
+    cache_dir = options.get("_", {}).get("cache_dir", "./cache")
```

Comment:
This is too much logic for individual scanners to need to know about in order to look up stuff in the cache directory. The domain-scan system should provide this as an easy utility method, global variable, or importable variable.